ChangeLog.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased changes
 
+## 0.26.1
+
+- Fix bug in transactionCost which expected a block hash header in response.
```

Comment:
```suggestion
- Fix bug in `transactionCost` caused by code that incorrectly assumed a block hash header to be present in a response.
```

Reply:
I don't think this level of detail is needed.